src/main/scala/com/yotpo/metorikku/output/writers/file/FileOutputWriter.scala

```diff
@@ -104,6 +107,14 @@ class FileOutputWriter(props: Map[String, Object], outputFile: Option[File]) ext
 
         writer.save()
 
+        fileOutputProperties.protectFromEmptyOutput match {
+          case Some(true) => {
+              val parquetFileDF = ss.read.parquet(path.get)
```

Comment:
I didn't check here ```path```, because I'm inside a matching case where pass must be passed

Reply:
It's not always a parquet.... you need to address the format